frontend/src/metabase/components/DriverWarning/DriverWarning.jsx

```diff
@@ -8,50 +8,67 @@ import {
   engineSupersedesMap,
 } from "metabase/entities/databases/forms";
 
-import Warnings from "metabase/query_builder/components/Warnings";
-
 import {
   CardContent,
   DriverWarningContainer,
   IconContainer,
 } from "./DriverWarning.styled";
+
 import ExternalLink from "metabase/components/ExternalLink";
+import Icon from "metabase/components/Icon";
 import MetabaseSettings from "metabase/lib/settings";
 
 const propTypes = {
   engine: PropTypes.string.isRequired,
+  hasCircle: PropTypes.bool.isRequired,
+  onChangeEngine: PropTypes.func.isRequired,
 };
 
 const driverUpgradeHelpLink = MetabaseSettings.docsUrl(
   "administration-guide/01-managing-databases",
 );
 
-function getSupersedesWarningContent(newDriver, supersedesDriver) {
+function getSupersedesWarningContent(
+  newDriver,
+  supersedesDriver,
+  onChangeEngine,
+) {
   return (
     <div>
       <p className="text-medium m0">
         {t`This is our new ${
           allEngines[newDriver]["driver-name"]
         } driver, which is faster and more reliable.`}
       </p>
-      <p>{t`The old driver has been deprecated and will be removed in the next release. If you really
-      need to use it, you can select ${
-        allEngines[supersedesDriver]["driver-name"]
-      } now.`}</p>
+
+      <p className="text-medium m0" style={{ marginTop: 8 }}>
+        {t`The old driver has been deprecated and will be removed in the next release. If you really
+      need to use it, you can `}
+        &nbsp;
+        <a
+          className="text-brand text-bold"
+          onClick={() => onChangeEngine(supersedesDriver)}
+        >{t`find it here`}</a>
+        .
+      </p>
     </div>
   );
 }
 
-function getSupersededByWarningContent(engine) {
+function getSupersededByWarningContent(engine, onChangeEngine) {
   return (
     <div>
       <p className="text-medium m0">
         {t`This driver has been deprecated and will be removed in the next release.`}
       </p>
-      <p className="text-medium m0">
-        {t`We recommend that you upgrade to the new ${
-          allEngines[engine]["driver-name"]
-        } driver, which is faster and more reliable.`}
+      <p className="text-medium m0" style={{ marginTop: 8, marginBottom: 8 }}>
```

Comment:
Tried adding this in the latest commit, but I must not have done it correctly since there is no margin added between paragraphs (see latest screenshots)